.github/workflows/e2e.yml

```
@@ -0,0 +1,31 @@
+name: e2e
+
+on:
+    pull_request:
+        types: [opened, synchronize]
+        paths: ['tests/e2e/**', 'lib/**', 'package.json', 'package-lock.json']
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
```

Let's use v4 now since Onyx is on Node 20:

```suggestion
            - uses: actions/checkout@v4
```